recipes/pyvipr/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyvipr" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 658ac1792c4e41293a9f23878e7f0e51c238eaa6bff16a503e08bc7aae2dc450
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed -vv "
+
+requirements:
+  host:
+    - ipywidgets >=7.0.0
```

Comment:
Are all of these packages required to build this package? Typically if they are not imported directly or indirectly in the `setup.py`, or listed under `setup_requires`, they aren't needed. It looks like you likely only need `python`, `pip`, and `jupyter-packaging`.  Please confirm that the rest are needed and if not remove them.

Reply:
Thanks for the feedback. I just deleted the unnecessary requirements

Reply:
I do not believe the ones listed under `run` are unnecessary because they appear to be actual runtime requirements. I was referring to the ones listed under `host` which were not needed to build the recipe.

Reply:
Got it. I just added back the requirements for `run`